orion/primitives/tadgan.py

```diff
@@ -46,34 +66,74 @@ class TadGAN(object):
             Shape of critic_x input.
         critic_z_input_shape (tuple):
             Shape of critic_z input.
-        layers_encoder (list):
-            List containing layers of encoder.
-        layers_generator (list):
-            List containing layers of generator.
-        layers_critic_x (list):
-            List containing layers of critic_x.
-        layers_critic_z (list):
-            List containing layers of critic_z.
         optimizer (str):
             String denoting the keras optimizer.
         learning_rate (float):
             Optional. Float denoting the learning rate of the optimizer. Default 0.005.
         epochs (int):
-            Optional. Integer denoting the number of epochs. Default 2000.
+            Optional. Integer denoting the number of epochs. Default 50.
         latent_dim (int):
             Optional. Integer denoting dimension of latent space. Default 20.
-        batch_size (int):
-            Integer denoting the batch size. Default 64.
         iterations_critic (int):
             Optional. Integer denoting the number of critic training steps per one
             Generator/Encoder training step. Default 5.
-        hyperparameters (dictionary):
+        shuffle (bool):
+            Whether to shuffle the dataset for each epoch. Default True.
+        validation_split (float):
+            Number between 0 and 1. Fraction of the training data to be used as validation data.
+            Default 0.2.
+        callbacks (tuple):
+            Callbacks to apply during training. Default tuple().
+        verbose (int):
+            Verbosity mode where 0 = silent, 1 = progress bar, 2 = one line per epoch. Default 0.
+        detailed_losses (bool):
+            Whether to output all loss values in verbose mode. Default True.
+        **hyperparameters (dict):
             Optional. Dictionary containing any additional inputs.
     """
 
+    def __init__(self, layers_encoder: list, layers_generator: list, layers_critic_x: list,
+                 layers_critic_z: list, optimizer: str, input_shape: Optional[tuple] = None,
+                 latent_shape: Optional[tuple] = None, target_shape: Optional[tuple] = None,
+                 latent_dim: int = 20, learning_rate: float = 0.005, epochs: int = 2000,
+                 batch_size: int = 64, iterations_critic: int = 5, shuffle: bool = True,
+                 callbacks: tuple = (), validation_ratio: float = 0.2,
```

Comment:
update `epochs` to 50 and `validation_ration` to 0.0 to be consistent with the json file, also check the docstrings above as well.

Reply:
updated